mina-daemon/templates/deployment.yaml

```diff
@@ -104,11 +119,8 @@ spec:
               ephemeral-storage: {{ .Values.resources.ephemeralStorageRequest }}
               {{- end }}
           image: {{ .Values.deployment.image }}
-          {{ if .Values.deployment.customEntrypoint.enabled -}}
-          command: [{{ .Values.deployment.customEntrypoint.entrypoint }}]
-          {{- end }}
```

Comment:
The custom entrypoint is no longer needed. This feature was never used as it default to `mina`

Reply:
at a time I saw entry points being mina and mina-dev, so I left it in case we will want to use mina-dev